stable/jenkins/templates/jenkins-test.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{.Release.Name}}-ui-test-{{ randAlphaNum 5 | lower }}"
+  annotations:
+    "helm.sh/hook": test-success
+    "pod.beta.kubernetes.io/init-containers": '[
```

Comment:
We can use the init-container field instead of the beta annotation now, as of 1.6.
I changed it in the [other chart](https://github.com/kubernetes/charts/pull/609/files) as well.